example/job_handlers/signal-app/__init__.py

```diff
@@ -21,30 +21,33 @@ def __init__(
     def _get_by_id(self, reference: str, depth: int = 1, attribute: str = ""):
         params = {"depth": depth, "attribute": attribute}
         req = requests.get(
-            f"{config.DMSS_API}/api/documents/{reference}?resolve_links=true&depth=100", params=params, headers=self.headers  # type: ignore
+            f"{config.DMSS_API}/api/documents/{reference}?resolve_links=true&depth=100", params=params,
+            headers=self.headers  # type: ignore
         )  # type: ignore
         req.raise_for_status()
         return req.json()
 
     def _update(self, reference: str, document: dict):
         form_data = {k: json.dumps(v) if isinstance(v, dict) else str(v) for k, v in document.items()}
         headers = {"Authorization": f"Bearer {self.job.token}", "Access-Key": self.job.token}
-        req = requests.put(f"{config.DMSS_API}/api/documents/{reference}", data=form_data, headers=headers, params={"update_uncontained": "False"})
+        req = requests.put(f"{config.DMSS_API}/api/documents/{reference}", data=form_data, headers=headers,
+                           params={"update_uncontained": "False"})
```

Comment:
What happened with the indentation?

Reply:
black creates a newline due to limit of line length is reached

Reply:
Right, it aligns with the parenthesis. Personally, I don't like that way of indenting, since the alignment would be off if you for instance rename the `requests` variable. And since you could do the renaming automatically, you might not notice the error.